src/main/java/io/wispforest/lavender/mixin/MinecraftClientMixin.java

```diff
@@ -43,4 +79,67 @@ private void onFrameEnd(boolean tick, CallbackInfo ci) {
         if (this.player == null) return;
         OffhandBookRenderer.endFrame();
     }
+
+    @Inject(method = "doItemPick", at = @At("HEAD"), cancellable = true)
```

Comment:
This feature, while interesting, shouldn't be part of this PR. It requires significantly too much non-trivial code that I don't currently have the capacity to review properly - it should be done as a separate PR later on

Reply:
alright, I'll split it out

Reply:
Moved to a separate PR.